lib/tasks/content_change_email.rake

```diff
@@ -0,0 +1,7 @@
+namespace :content_change_email do
```

Comment:
I think it might be worth putting this inside a `reports` namespace.

Reply:
👍